model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -234,6 +258,8 @@ class ModelReplicationServer(
         repository: String,
         lastKnown: String?,
     ) {
+        checkPermission("repository", repository, "objects", "read")
+
         // TODO permission check on the repository ID is not sufficient, because the client could
```

Comment:
This comment is only true for objects stored in legacy storage.

Reply:
The comment is just outdated. Removed it.